user-storage-jpa/src/main/java/org/keycloak/quickstart/storage/user/EjbExampleUserStorageProvider.java

```diff
@@ -217,7 +217,7 @@ public boolean isValid(RealmModel realm, UserModel user, CredentialInput input)
         if (!supportsCredentialType(input.getType()) || !(input instanceof UserCredentialModel)) return false;
         UserCredentialModel cred = (UserCredentialModel)input;
         String password = getPassword(user);
-        return password != null && password.equals(cred.getValue());
+        return password != null && password.equals(cred.getChallengeResponse());
```

Comment:
Remove this - we need to fix API issues on the server side

Reply:
I sent PR to fix this on server side. If you have a chance to review, it is here keycloak/keycloak#6795

Reply:
Resolving this comment, see https://github.com/keycloak/keycloak-quickstarts/pull/166#discussion_r380572963